aikau/src/main/resources/alfresco/menus/AlfMenuBarPopup.js

```diff
@@ -109,32 +109,51 @@ define(["dojo/_base/declare",
          {
             this.title = this.message(this.title);
          }
+         if (!this.iconAltText && this.label)
+         {
+            this.iconAltText = this.label;
+         }
+         else
+         {
+            this.iconAltText = this.message(this.iconAltText);
+         }
```

Comment:
It looks like `this.label` won't be "messaged"? Could this just be written as `this.iconAltText = this.message(this.iconAltText || this.label);`?